lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2611,3 +2613,25 @@ definitions:
         pattern: '^(\d{6})$'
         example: '176148'
         x-nullable: false
+
+  paginated_enodebs:
+    description: Page of enodebs
```

Comment:
Nit: be consistent about whether you're calling them "eNodeBs" (full/correct capitalization) or "enodebs" (simple/economical capitalization)